modules/gitops-storing-and-retrieving-argo-cd-logs.adoc

```diff
@@ -10,8 +10,8 @@ You can use the Kibana dashboard to store and retrieve Argo CD logs.
 
 .Prerequisites
 
-* The {gitops-title} Operator is installed in your cluster.
-* The {logging-title} is installed with default configuration in your cluster.
+* You have installed the {gitops-title} Operator on your {OCP} cluster.
+* The {logging-title} is installed with default configuration on your {OCP} cluster.
```

Comment:
Same here as earlier, keep this parallel if possible

Reply:
Fixed.